### PR TITLE
[KSQL] Avro to Protobuf tutorial update

### LINKED ISSUE #369: [KSQL] Change serialization format from Avro to Proto
Extend existing tutorial to demo json -> avro -> protobuf format change.

### PR DESCRIPTION
**Pull request by gAmUssA**
Updating existing tutorial to demo avro to protobuf migration
FIXES #369﻿

while it's hot :-P

tenor-141673481

### CODE REVIEW

**Comment by gAmUssA** [comment]
I temporarily disabled KStreams example. I will rewrite it with Avro to proto. 
The issue for tracking #376

**Comment by colinhicks** [comment]
Thanks @gAmUssA. It's cool to see protobuf in action here. Do you plan on a separate, stacked PR to re-enable the Kafka Streams flavor? And do you think we should review this PR, but hold off on merging to master until a PR for #376 is ready?

**Comment by bbejeck** [comment]
@gAmUssA 
For me, I'd like to get at least 1 Kafka 101 tutorial merged (hint #339 needs review love)

\cc @colinhicks

File: _data/tutorials.yaml [hunk 1]
```diff
@@ -162,10 +162,10 @@ serialization:
   meta-description: "Learn to convert a stream's serialization format"
   slug: "/changing-serialization-format"
   problem: "you have a Kafka topic with the data serialized in a particular format, and you want to change the format to something else."
-  introduction: "Consider a topic with events that represent movie releases. The events in the topic are formatted with JSON. In this tutorial, we'll write a program that creates a new topic with the same events, but formatted with Avro."
+  introduction: "Consider a topic with events that represent movie releases. The events in the topic are formatted with Avro. In this tutorial, we'll write a program that creates a new topic with the same events, but formatted with Protobuf."
   status:
     ksql: enabled
-    kstreams: enabled
+    kstreams: enable
```
Comment (bbejeck) [inline _data/tutorials.yaml:168]: needs to be `enabled`

Comment (gAmUssA) [reply _data/tutorials.yaml]: ✅ done

**Pull request merged**